src/_test_/home.test.js

```diff
@@ -0,0 +1,9 @@
+import { render } from '@testing-library/react';
+import HomePage from '../components/home';
+
+describe('test HomePage component', () => {
+  test('it renders correctly', () => {
+    const tree = render(<HomePage />);
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] All tests should be passed. Kindly work on the failed test. :+1: 

![test1](https://user-images.githubusercontent.com/70483835/215102003-3146b96b-6478-49a3-bbd4-09f929385361.png)